documentation/Glossary.md

```diff
@@ -72,6 +76,38 @@ An input to be processed by a [CustomDataProcessor](#customdataprocessor). In mo
 
 A [Table](#table) that is built outside of the standard [table building cycle](#table-building-cycle). DynamicTables are commonly built in response to a [TableCommand](#tablecommand) being invoked.
 
+## Pivot Column
+
+NOTE: (Not to be confused with the [Special Column](#special-columns) `PivotColumn`)
+
+A special type of [Column](#column) that should be pivoted about when its [Table](#table) is interpreted as a [Pivot Table](#pivot-table). Any Column that appears in a [TableConfiguration](#tableconfiguration) before the `PivotColumn` is a Pivot Column.
+
+## Pivot Table
+
+A representation of a [Table](#table) where certain [Columns](#column) are pivoted/grouped about. For example, consider this table with 5 rows:
+
+| State        | Zip Code | Population |
+|--------------|----------|------------|
+| Washington   | 98052    | 65,558     |
+| New York     | 10023    | 60,998     |
+| Washington   | 98101    | 13,877     |
+| Washington   | 98109    | 20,715     |
+| New York     | 10025    | 94,600     |
+
+If the "State" column above is a [Pivot Column](#pivot-column), the 5 rows would be grouped into the following Pivot Table:
+
+| State        | Zip Code | Population |
+|--------------|----------|------------|
+| > Washington |          |            |
+|              | 98052    | 65,558     |
+|              | 98101    | 13,877     |
+|              | 98109    | 20,715     |
+| > New York   |          |            |
+|              | 10023    | 60,998     |
+|              | 10025    | 94,600     |
+
+NOTE: the SDK has no understanding of Pivot Tables. Tables created by a plugin are purely "flat" tables - i.e. tables similar the first one above. It is up to programs like [Windows Performance Analyzer](#windows-performance-analyzer) to use pivot information in a [Table Configuration](#tableconfiguration) to present a plugin's Table as a Pivot Table.
```

Comment:
Not sure if just saying "The UI layer" suffices, since there need-not be a UI later. For example, one could make a program on-top of the Engine that has no UI but understands how to pivot things.